### DISCUSSION TITLE
License question

### DISCUSSION THREAD

**Discussion by danylo-kelvich** [open-mmlab/mmclassification#1563] [General] [answered]
Hi! I see that this repo has Apache License 2.0, but some of the models here were originally developed under a non-commercial projects. So my question is - can I use any model from mmpretrain for commercial projects?

**Comment by Ezra-Yu** [comment, marked as answer]
If weight is provided by mmpretrain training, then yes; if it sources a third-party library (weight with *), then it still needs to follow its original protocol.
